native/src/modules/endpoint/hocs/withPayloadProvider.js

```diff
@@ -112,13 +113,12 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R: {},
         return <LanguageNotAvailableContainer languages={props.availableLanguages}
                                               changeLanguage={changeUnavailableLanguage} />
       } else if (props.status === 'loading') {
-        console.log('render loading', props.progress)
         return timeoutExpired
-          ? <LayoutedScrollView refreshControl={<RefreshControl refreshing />}>
+          ? <LayoutedScrollView refreshControl={<RefreshControl refreshing={false} />}>
```

Comment:
I think we still want to have the refreshing if the ProgressContainer is not shown, i.e. if there are innerProps and dispatch.